src/mgr/CMakeLists.txt

```diff
@@ -50,5 +52,8 @@ if(WITH_MGR)
     ${GSSAPI_LIBRARIES})
   set_target_properties(ceph-mgr PROPERTIES
     POSITION_INDEPENDENT_CODE ${EXE_LINKER_USE_PIE})
+  if(WITH_LTTNG)
+    add_dependencies(ceph-mgr mgr_op_tp)
+  endif()
```

Comment:
@pdvian there is a build failure likely related to this part of the PR:
https://shaman.ceph.com/builds/ceph/wip-yuri4-testing-2024-02-03-0802/eec7af66764e6dd95c56165a0802a6f1f3a302ae/default/380003/
```
error: Installed (but unpackaged) file(s) found:
   /usr/lib/debug/usr/lib64/libmgr_op_tp.so.1.0.0-19.0.0-1254.geec7af66.el8.aarch64.debug
   /usr/lib64/libmgr_op_tp.so
   /usr/lib64/libmgr_op_tp.so.1
   /usr/lib64/libmgr_op_tp.so.1.0.0
```

Reply:
Let me check it today. Thanks!